test/test_rosbag/bag_migration_tests/README.md

```diff
@@ -0,0 +1,349 @@
+# Bag migration tests scripts
+
+In order to generate the data and rules for bag migration tests, two scripts are provided:
+
+ * **generate_data** : Generates *.bag* files with messages (data) to run the bag migration tests. 
+ * **generate_rules** : Help in the process of creating rules between two message generations.
+
+The next sections explain how to use these scripts in detail. The rest are auxiliary scripts that should NOT be used directly.
+
+Before running any of these scripts you need to build the **test_rosbag** package at least once in your workspace and `source` the environment workspace setup file. This can be done running the following from the root of the workspace:
+
+``` bash
+$ catkin_make
+$ source devel/setup.bash
```

Comment:
Never mind, I get it to work. This is what I did wrong:

1. I call `catkin_make_isolated` with `--pkg` from a clean ws. And according with the documentation I need to call `cakin_make_isolated` w/o `--pkg` once first.
``` bash
  --pkg PKGNAME [PKGNAME ...]
                        Invoke 'make' on specific packages (only after
                        catkin_make_isolated has been invoked before with the
                        same install flag)
```

Otherwise I have an error saying that `cmake` isn't found:

``` bash
==> cmake /home/eperdomo/dev/ws/clearpath_ws/src/ros_comm/test/test_rosbag -DCATKIN_DEVEL_PREFIX=/home/eperdomo/dev/ws/clearpath_ws/devel_isolated/test_rosbag -DCMAKE_INSTALL_PREFIX=/home/eperdomo/dev/ws/clearpath_ws/install_isolated -G Unix Makefiles in '/home/eperdomo/dev/ws/clearpath_ws/build_isolated/test_rosbag'
Unhandled exception of type 'OSError':
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/catkin/builder.py", line 965, in build_workspace_isolated
    number=index + 1, of=len(ordered_packages)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/catkin/builder.py", line 665, in build_package
    destdir=destdir, use_ninja=use_ninja
  File "/opt/ros/indigo/lib/python2.7/dist-packages/catkin/builder.py", line 397, in build_catkin_package
    run_command_colorized(cmake_cmd, build_dir, quiet, add_env=add_env)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/catkin/builder.py", line 187, in run_command_colorized
    run_command(cmd, cwd, quiet=quiet, colorize=True, add_env=add_env)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/catkin/builder.py", line 205, in run_command
    raise OSError("Failed command '%s': %s" % (cmd, e))
OSError: Failed command '['/home/eperdomo/dev/ws/clearpath_ws/devel_isolated/rosbag/env.sh', 'cmake', '/home/eperdomo/dev/ws/clearpath_ws/src/ros_comm/test/test_rosbag', '-DCATKIN_DEVEL_PREFIX=/home/eperdomo/dev/ws/clearpath_ws/devel_isolated/test_rosbag', '-DCMAKE_INSTALL_PREFIX=/home/eperdomo/dev/ws/clearpath_ws/install_isolated', '-G', 'Unix Makefiles']': [Errno 2] No such file or directory
<== Failed to process package 'test_rosbag': 
  Failed command '['/home/eperdomo/dev/ws/clearpath_ws/devel_isolated/rosbag/env.sh', 'cmake', '/home/eperdomo/dev/ws/clearpath_ws/src/ros_comm/test/test_rosbag', '-DCATKIN_DEVEL_PREFIX=/home/eperdomo/dev/ws/clearpath_ws/devel_isolated/test_rosbag', '-DCMAKE_INSTALL_PREFIX=/home/eperdomo/dev/ws/clearpath_ws/install_isolated', '-G', 'Unix Makefiles']': [Errno 2] No such file or directory
Command failed, exiting.
```

2. I was trying to pass `--only-pkg-with-deps`, which as you're saying I don't need. In fact, I can now use `--pkg`.
``` bash
  --only-pkg-with-deps ONLY_PKG_WITH_DEPS [ONLY_PKG_WITH_DEPS ...]
                        Only consider the specific packages and their
                        recursive dependencies and ignore all other packages
                        in the works
```

With this one the error was a missed dependency.

Either way, I've just updated the PR to use `catkin_make_isolated`.